kodev

```diff
@@ -399,6 +399,36 @@ TARGET:
 }
 
 
+function kodev-copy {
+    LOG_HELP_MSG="
+usage: copy <Root folder of KOReader on target device>
+
+Example: ./kodev copy /media/hzj_jie/HZJ-VOYAGE/koreader
+    "
+
+    [ $# -lt 1 ] && { echo "${LOG_HELP_MSG}"; exit 1; }
+
+    case $1 in
+        -h | --help)
+            echo "${LOG_HELP_MSG}" ;;
+        *)
+            for i in $(git diff upstream/master --stat |
+                       grep '|' |
+                       sed 's/ /\t/g' |
+                       cut -f 2 |
+                       grep -v "^kodev$")
+            do
+                cp $i $1/$i
```

Comment:
If I understand you correctly, you mean rsync from koreader-kindle-arm-linux-gnueabi/ to the device. The major disadvantage is you need to select device type.
Meanwhile, you always need to build KOReader, which seems not necessary for Lua developers. i.e. You do not need to build KOReader at all, but changed Lua files can be easily pushed to your device with ./kodev copy {device-folder}.

I have no strong opinion, if you would like to avoid this change, I can revert it.